app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use Exception;
+
+class UserController extends Controller
+{
+    public function edit($id)
+    {
+        $user = User::findOrFail($id);
+
+        return view('user.user-info', ['user' => $user]);            
+    }
+
+    public function update(Request $request, $id)
+    {
+        $data = $request->only([
+            'fullname',
+            'nickname',
+            'image',
+        ]);
+
+        if (isset($data['image'])) {
+            $uploaded = $this->upload($data['image']);
+
+            if (!$uploaded['status']) {
+                return back()->with('status', $uploaded['msg']);
+            }
+
+            $data['image'] = $uploaded['file_name'];
+        }
+
+        try {
+            $user = User::find($id);
+            $user->update($data);   
+        } catch (\Exception $e) {
```

Comment:
```suggestion
        } catch (Exception $e) {
```
ở đầu file đã `use Exception` rồi nên ko cần `\` nữa